lnrpc/rpc.proto

```diff
@@ -1744,6 +1744,8 @@ message NodeInfo {
 
     uint32 num_channels = 2 [json_name = "num_channels"];
     int64 total_capacity = 3 [json_name = "total_capacity"];
+
```

Comment:
could use a comment here so that it is automatically updated in our documentation.

if you don't mind, could you also add docs for `num_channels` and `total_capacity` while we're here? :)